doc/api/net.md

```diff
@@ -266,6 +266,9 @@ added: v0.11.14
     for all users. **Default:** `false`
   * `writableAll` {boolean} For IPC servers makes the pipe writable
     for all users. **Default:** `false`
+  * `ipv6Only` {boolean} For TCP servers, setting `ipv6Only` to `true` will
+    disable the dual-stack support, i.e., binding host `::` won't make
```

Comment:
Just "disable dual-stack support", no "the." (And you'd normally say "binding to host", not "binding host".)